flink-streaming-connectors/flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/container/RedisContainer.java

```diff
@@ -68,12 +68,12 @@ public RedisContainer(final JedisSentinelPool sentinelPool) {
 	 */
 	@Override
 	public void close() throws IOException {
-		if (this.jedisPool != null) {
-			this.jedisPool.close();
-		}
-		if (this.jedisSentinelPool != null) {
-			this.jedisSentinelPool.close();
-		}
+		getInstance().close();
+	}
+
+	@Override
+	public void open() throws Exception {
+		getInstance().echo("Test");
```

Comment:
Yes. It tries to open a connection and just echo the message "Test". 
This is to see if there is a way to communicate with the cluster or server (depending on the setup).